packages/firestore/src/protos/firestore_proto_api.ts

```diff
@@ -224,6 +224,15 @@ export declare namespace firestoreV1ApiClientInterfaces {
   interface ExistenceFilter {
     targetId?: number;
     count?: number;
+    unchangedNames?: BloomFilter;
+  }
+  interface BloomFilter {
+    bits?: BitSequence;
+    hashCount?: number;
+  }
+  interface BitSequence {
+    bitmap?: string;
```

Comment:
In proto, bitmap is bytes. 
Should I write the type as `string`, `Uint8Array`, or `string | Uint8Array`